pyartcd/pyartcd/util.py

```diff
@@ -310,3 +310,232 @@ async def sync_images(version: str, assembly: str, operator_nvrs: list,
             doozer_data_path=doozer_data_path,
             doozer_data_gitref=doozer_data_gitref
         )
+
+
+def default_release_suffix():
+    """
+    Returns a release suffix based on current timestamp
+    E.g. "201901011200.?
```

Comment:
201901011200.? -> 201901011200.p?

Reply:
Fixed in 047f1e932b5cb53b464f24227ee324c30a9caec0